executor/api/rest/client_test.go

```diff
@@ -266,3 +269,32 @@ func TestClientMetrics(t *testing.T) {
 	}
 
 }
+
+func TestErrorResponse(t *testing.T) {
+	t.Logf("Started")
+	g := NewGomegaWithT(t)
+	h := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(errorPredictResponse))
+	})
+	host, port, httpClient, teardown := testingHTTPClient(g, h)
+	defer teardown()
```

Comment:
Gomega's BeforeEach and AfterEach are another option for how to manage test setup and teardown

Reply:
As its not for all test might leave for now?